### PR TITLE
feat: add login-signup page

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added Signup & Login Pages**
> Signup Page
> Login Page

signup-page

login-page

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]
Really Good UI design!

File: pages/login-signup/signup.html [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                <div class="overlay">
+                    <div class="overlay__panel overlay__left">
+                        <h1>Welcome Back!</h1>
+                        <p>
+                            To keep connected with us please login with your
+                            personal info
+                        </p>
+                    </div>
+                </div>
+            </div> -->
```
Comment (tewarimohit) [inline pages/login-signup/signup.html]: Is there any specific reason for commenting this section...

**Review by harshparab — APPROVED** [review]

**Review by tewarimohit — COMMENTED** [review]
Good to merge.. Remove commented part if not necessary...

File: pages/login-signup/signup.html (hunk 1 above)
Comment (nerdyspook) [reply pages/login-signup/signup.html]: I was trying a different design initially but couldn't think of making it responsive.

**Pull request merged**